res/skins/LateNight/style_palemoon.qss

```diff
@@ -1736,80 +1736,96 @@ WPushButton#RecButton[displayValue="1"],
 #Hotcue1 WPushButton[displayValue="0"] {
   image: url(skin:/palemoon/buttons/btn__1.svg) no-repeat center center;
   }
-  #Hotcue1 WPushButton[displayValue="1"] {
+  #Hotcue1 WPushButton[displayValue="1"],
+  #Hotcue1 WPushButton[displayValue="2"] {
```

Comment:
This should probably be backported to 2.3.

Reply:
Why? Without this PR the display value will never be 2, because the "active" state does not exist.

Reply:
oh, nevermind then